components/deployment-synchronizer/org.wso2.carbon.deployment.synchronizer/src/main/java/org/wso2/carbon/deployment/synchronizer/repository/CarbonRepositoryUtils.java

```diff
@@ -23,11 +23,7 @@
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.wso2.carbon.base.ServerConfiguration;
-import org.wso2.carbon.deployment.synchronizer.ArtifactRepository;
-import org.wso2.carbon.deployment.synchronizer.DeploymentSynchronizerException;
-import org.wso2.carbon.deployment.synchronizer.internal.DeploymentSynchronizationManager;
-import org.wso2.carbon.deployment.synchronizer.internal.DeploymentSynchronizer;
-import org.wso2.carbon.deployment.synchronizer.internal.DeploymentSynchronizerConstants;
+import org.wso2.carbon.deployment.synchronizer.*;
```

Comment:
This may be due to Intellij Idea configuration, Better you customize it.

Thanks,
Shameera.

On Sun, Feb 22, 2015 at 12:36 PM, Kasun Gajasinghe <notifications@github.com
> wrote:

> In
> components/deployment-synchronizer/org.wso2.carbon.deployment.synchronizer/src/main/java/org/wso2/carbon/deployment/synchronizer/repository/CarbonRepositoryUtils.java
> <https://github.com/wso2/carbon-commons/pull/52#discussion_r25134234>:
>
> > @@ -23,11 +23,7 @@
> >  import org.apache.commons.logging.Log;
> >  import org.apache.commons.logging.LogFactory;
> >  import org.wso2.carbon.base.ServerConfiguration;
> > -import org.wso2.carbon.deployment.synchronizer.ArtifactRepository;
> > -import org.wso2.carbon.deployment.synchronizer.DeploymentSynchronizerException;
> > -import org.wso2.carbon.deployment.synchronizer.internal.DeploymentSynchronizationManager;
> > -import org.wso2.carbon.deployment.synchronizer.internal.DeploymentSynchronizer;
> > -import org.wso2.carbon.deployment.synchronizer.internal.DeploymentSynchronizerConstants;
> > +import org.wso2.carbon.deployment.synchronizer.*;
>
> Hi Manoj,
>
> Can you remove this star package import please?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wso2/carbon-commons/pull/52/files#r25134234>.
>



-- 
Best Regards,
Shameera Rathnayaka.

email: shameera AT apache.org , shameerainfo AT gmail.com
Blog : http://shameerarathnayaka.blogspot.com/